modules/juce_graphics/native/juce_mac_CoreGraphicsHelpers.h

```diff
@@ -63,6 +63,9 @@ CGContextRef juce_getImageContext (const Image&);
 #endif
 
 #if JUCE_MAC
+ Colour juce_convertColourToDisplayColourSpace (const Colour& colour,

```

Comment:
How about `convertSRGBTo...` instead? IIUC this more accurately describes what this does

Reply:
`colour` is redundant in the declaration